les/protocol.go

```diff
@@ -34,17 +34,18 @@ import (
 const (
 	lpv2 = 2
 	lpv3 = 3
+	lpv4 = 4
```

Comment:
Should we bump the protocol version in this PR?

I would suggest to bump the les protocol in another separate PR with all changes included.

Reply:
removed lpv4 from the supported protocol lists